source/brailleDisplayDrivers/hims.py

```diff
@@ -190,12 +190,13 @@ def registerAutomaticDetection(cls, driverRegistrar: bdDetect.DriverRegistrar):
 		# Bulk devices
 		driverRegistrar.addUsbDevices(bdDetect.DeviceType.CUSTOM, {
 			"VID_045E&PID_930A",  # Braille Sense & Smart Beetle
-			"VID_045E&PID_930B",  # Braille EDGE 40
+   			"VID_045E&PID_930B",  # Braille EDGE 40

```

Comment:
Please undo the addition of spaces

```suggestion
			"VID_045E&PID_930B",  # Braille EDGE 40
```